build.sbt

```diff
@@ -900,14 +900,14 @@ lazy val sandbox = projectMatrix
     // Ignore deprecation warnings here - it's all generated code, anyway.
     scalacOptions ++= Seq(
       "-Wconf:cat=deprecation:silent"
-    ) ++ scala3MigrationOption(scalaVersion.value),
```

Comment:
I can't remember what this was doing now but I think it started causing a problem, and didn't seem to be necessary.